monkestation/code/modules/art_sci_overrides/artifact_components/bonk.dm

```diff
@@ -0,0 +1,38 @@
+/datum/component/artifact/bonk
+	associated_object = /obj/structure/artifact/bonk
+	weight = ARTIFACT_UNCOMMON
```

Comment:
Ive been seeing mass borg in far too many rounds, this should be more rare.

Reply:
thats the bonk artifact not borger

Reply:
I did a dumb.